_utils/spellcheck.sh

```diff
@@ -0,0 +1,152 @@
+#!/bin/bash
+
+set -e
+set -o pipefail
+
+v() { [ -n "$VERBOSE" ] && echo "$@" ; }
+
+if command -v hunspell > /dev/null; then
+    check_words() { hunspell -l; }
+elif command -v aspell > /dev/null; then
+    check_words() { aspell | grep '^[#&]' | cut -d' ' -f2; }
+else
+    echo "${0##*/}: No spell checker installed. Need hunspell or aspell." >&2
+    exit 1
+fi
+
+extract_words() {
+    pandoc -tjson "$@" | jq -r '
```

Comment:
Ah, there is `make` below. So, it is supposed to run specifically from this repository for having `Makefile` (not any of its submodules), and will compare existing commits.
In that case, pull requests to submodules (`qubes-doc`) will be a problem, because:
a) you don't update submodules
b) at this point there may be no commit pointing to new submodule revision yet

So, maybe better expect (alternatively?) already built _site directories for old and new versions? `_utils/travis.sh` already create them.